spec/services/subscriptions/dates_service_spec.rb

```diff
@@ -19,7 +19,7 @@
   let(:pay_in_advance) { false }
 
   let(:subscription_at) { DateTime.parse('02 Feb 2021') }
-  let(:billing_date) { DateTime.parse('07 Mar 2022') }
+  let(:billing_date) { Time.parse('2022-03-07 04:20:46.011') }
```

Comment:
Rails/TimeZone: Do not use `Time.parse` without zone. Use one of `Time.zone.parse`, `Time.current`, `Time.parse.in_time_zone`, `Time.parse.utc`, `Time.parse.getlocal`, `Time.parse.xmlschema`, `Time.parse.iso8601`, `Time.parse.jisx0301`, `Time.parse.rfc3339`, `Time.parse.httpdate`, `Time.parse.to_i`, `Time.parse.to_f` instead. (https://rails.rubystyle.guide#time, http://danilenko.org/2012/7/6/rails_timezones)